extensions/amp-story/1.0/amp-story-reaction-quiz.css

```diff
@@ -14,9 +14,11 @@
  * limitations under the License.
  */
 
-@import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
-
 .i-amphtml-story-reaction-quiz-container {
+    --reaction-prompt-text-color: white !important;
+    --reaction-chip-shadow: none !important;
+    --reaction-answer-choice-border: var(--reaction-accent-color) !important;
+    --reaction-answer-choice-background: white !important;
```

Comment:
Splitting the options between quiz and reaction looks like a step in the wrong direction if our goal is consistency across components.